src/views/notifications/Histories.vue

```diff
@@ -0,0 +1,254 @@
+                                {{ notification.user.name }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="absolute right-0 top-0 mr-5 mt-3 dropdown" v-if="permissions.notifications_update || permissions.notifications_get_single">
+                            <a
+                              class="dropdown-toggle w-5 h-5 block"
+                              href="javascript:;"
+                              aria-expanded="false"
+                            >
+                              <MoreHorizontalIcon class="w-5 h-5 text-gray-600 dark:text-gray-300"/>
+                            </a>
+                            <div class="dropdown-menu w-40">
+                              <div class="dropdown-menu__content box dark:bg-dark-1 p-2">
+                                <a href="javascript:;" v-if="permissions.notifications_get_single" @click="this.$router.push({ name: 'moderation.notifications.view', params: { id: notification.id } })" data-dismiss="dropdown" class="flex items-center block p-2 transition duration-300 ease-in-out bg-white dark:bg-dark-1 hover:bg-gray-200 dark:hover:bg-dark-2 rounded-md">
+                                  <EyeIcon class="w-4 h-4 mr-2" /> View
+                                </a>
+                                <a href="javascript:;" v-if="permissions.notifications_update" @click="this.$router.push({ name: 'moderation.notifications.edit', params: { id: notification.id } })" data-dismiss="dropdown" class="flex items-center block p-2 transition duration-300 ease-in-out bg-white dark:bg-dark-1 hover:bg-gray-200 dark:hover:bg-dark-2 rounded-md">
+                                  <Edit2Icon class="w-4 h-4 mr-2" /> Edit
+                                </a>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                        <div class="text-center lg:text-left px-5 pb-5">
+                          <div class="flex">
+                            <!-- BEGIN: Notification Seen -->
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5 self-center mr-3" v-if="notification.seen">
+                              <EyeIcon class="w-3 h-3 mr-2"/>Seen
+                            </div>
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5 self-center mr-3" v-else>
+                              <EyeOffIcon class="w-3 h-3 mr-2"/>Unseen
+                            </div>
+                            <!-- END: Notification Seen -->
+                            <!-- BEGIN: Notification Type -->
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5 self-center" v-if="notification.target_id !== null">
```

Comment:
Unnecessary `!== null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/7ce1e76f3284139350f2ec7415415ff6b695b571